service/service_construction.go

```diff
@@ -588,3 +621,176 @@ func (s ConstructionService) ConstructionSubmit(
 		},
 	}, nil
 }
+
+func (s ConstructionService) CreateOperationDescription(
+	operations []*types.Operation,
+) ([]*parser.OperationDescription, error) {
+	if len(operations) != 2 { //nolint:gomnd
+		return nil, fmt.Errorf("invalid number of operations")
```

Comment:
Error should be a variable

Reply:
It's an invalid input error, it's wrapped in the function calling this one (same with the others).  Doesn't seem like we need a unique error type for every way the input could be invalid

Reply:
Can't find my previous comment, but the caller is wrapping all the errors you mentioned within an invalidInputErr.  We don't want unique errors defined for each way that the input can be incorrect